SeleniumTrainingCenter/PageObjects/Interfaces/IWishListPage.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SeleniumTrainingCenter.PageObjects.Interfaces
+{
+    public interface IWishListPage
```

Comment:
I think you're huge fan of using interfaces:) This is no bad, but I assume that some of these interfaces could be skipped